src/launcher/features/apps/appsEffects.ts

```diff
@@ -160,35 +159,42 @@ const install =
     };
 
 export const installDownloadableApp =
-    (app: DownloadableApp, version?: string): AppThunk =>
+    (app: DownloadableApp, toVersion?: string): AppThunk =>
     dispatch => {
-        if (version == null) {
-            sendAppUsageData(EventAction.INSTALL_APP, app.source, app.name);
-        } else {
-            sendAppUsageData(
-                EventAction.INSTALL_APP_OLD_VERSION,
-                app.source,
-                `${app.name} (${version})`
-            );
-        }
+        sendAppUsageData(EventAction.APP_MANAGEMENT, {
+            action:
+                toVersion !== app.latestVersion
+                    ? 'Install Explicit Version'
+                    : 'Install',
+            ...app,
+            toVersion: toVersion ?? app.latestVersion,
+        });
 
         dispatch(installDownloadableAppStarted(app));
-        return dispatch(install(app, version));
+        return dispatch(install(app, toVersion));
     };
 
 export const updateDownloadableApp =
     (app: DownloadableApp): AppThunk =>
     dispatch => {
-        sendAppUsageData(EventAction.UPDATE_APP, app.source, app.name);
+        sendAppUsageData(EventAction.APP_MANAGEMENT, {
+            action: 'Update',
+            ...app,
+            toVersion: app.latestVersion,
```

Comment:
Same here, is there a reason why it's called `toVersion`, can't it just be called `version`?

edit:
I'm starting to see why, so in the telemetric you want the apps to have both `toVersion` and `fromVersion` in order detect what versions users are updating to.
Then, do you not want the `updateDownloadableApp` to have a `fromVersion` as well as a `toVersion`?